src/halmos/__main__.py

```diff
@@ -537,18 +543,21 @@ def run(
 
     # return test result
     exitcode = 0 if passed else 1
-    num_counterexamples = sum(m.result == sat for m in models)
+    sat_models = [m.model for m in models if m.result == sat]
```

Comment:
this includes potentially invalid models even if print_potential_counterexample is not set true.

Reply:
is that a problem for json output?

Reply:
yeah it is, fixed in b5edbd1